.github/scripts/generate-docs-workflow.psm1

```diff
@@ -0,0 +1,38 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseSingularNouns', '', Scope='Function')]
```

Comment:
I'd much rather see the function name change than suppressing the message. Something like `Format-AutoGeneratedMarkdown` or even just dropping the "s": `Format-AutoGeneratedMarkdownFile` would work.

It's convention that the noun is always singular for PS.

Reply:
fair enough 👍 renamed the function.